openlibrary/macros/WorkCarousel.html

```diff
@@ -0,0 +1,5 @@
+$def with(work)
+
+$if work:
+    $:render_template("home/custom_ia_carousel", title="You might also like:", key="related-subjects-carousel", work_id=work.key, _type="subjects", limit=42)
+    $:render_template("home/custom_ia_carousel", title="More by this author:", key="related-authors-carousel", work_id=work.key, _type="authors", limit=42)
```

Comment:
If it's not too much work, can we display "More works by H. G. Wells" instead? I think it looks a wee bit nicer is a little clearer to the user.

Reply:
What if there are `n` authors? Display them all? Choose the first?

Reply:
Oooh, good point, I didn't think of that. The query is `OR`ing all the authors, so I guess display them all? Or we could display the name if it's only one, and then say "More from these authors" if there are more. It looks like goodreads, which has a similar feature, just uses the first author: https://www.goodreads.com/book/show/234731.Invasive_Procedures